newrelic/hooks/external_botocore.py

```diff
@@ -218,7 +218,16 @@ def extract_bedrock_titan_embedding_model_request(request_body, bedrock_attrs):
     return bedrock_attrs
 
 
-def extract_bedrock_titan_embedding_model_response(response_body, bedrock_attrs):
+def extract_bedrock_cohere_embedding_model_request(request_body, bedrock_attrs):
+    request_body = json.loads(request_body)
+
+    try:
+        input_text = str(request_body.get("texts"))
```

Comment:
I'm not following this try catch. It tries to cast it to a string, otherwise it doesn't and it gets cast to a string later by our record custom event attribute validation. Seems like we could simplify this to just:
```
input_text = request_body.get("texts")
```